package.json

```diff
@@ -11,7 +11,7 @@
     "prettier": "prettier --write .",
     "prepare": "husky install",
     "pre-commit": "lint-staged",
-    "test": "npm run test:db:reset && node --inspect node_modules/.bin/jest --runInBand --watch --testNamePattern",
+    "test": "jest",
```

Comment:
why did you change the default test script?

Reply:
please always add a comment to the PR for such changes (difficult to understand for reviewers what you did there). this will save us time